apiserver/facades/controller/caasunitprovisioner/provisioner_test.go

```diff
@@ -189,15 +189,14 @@ func (s *CAASProvisionerSuite) TestApplicationConfig(c *gc.C) {
 	c.Assert(results.Results[1].Error, jc.DeepEquals, &params.Error{
 		Message: `"unit-gitlab-0" is not a valid application tag`,
 	})
-	c.Assert(results.Results[0].Config, jc.DeepEquals, map[string]interface{}{"foo": "bar", "juju-managed-units": false})
+	c.Assert(results.Results[0].Config, jc.DeepEquals, map[string]interface{}{"foo": "bar"})
 }
 
 func strPtr(s string) *string {
 	return &s
 }
 
 func (s *CAASProvisionerSuite) TestUpdateApplicationsUnitsNoTags(c *gc.C) {
```

Comment:
The tags vs no tags logic would all need to be fixed if we remove the juju-managed-units config from the backend entirely